apisix/discovery/consul/init.lua

```diff
@@ -176,80 +190,304 @@ local function get_retry_delay(retry_delay)
 end
 
 
+local function get_opts(consul_server, is_catalog)
+    local opts = {
+        host = consul_server.host,
+        port = consul_server.port,
+        connect_timeout = consul_server.connect_timeout,
+        read_timeout = consul_server.read_timeout,
+    }
+    if consul_server.keepalive then
+        if is_catalog then
+            opts.default_args = {
+                wait = consul_server.wait_timeout, --blocked wait!=0; unblocked by wait=0
+                index = consul_server.catalog_index,
+            }
+        else
+            opts.default_args = {
+                wait = consul_server.wait_timeout, --blocked wait!=0; unblocked by wait=0
+                index = consul_server.health_index,
+            }
+        end
+    end
+
+    return opts
+end
+
+
+local function watch_catalog(consul_server)
+    local client = resty_consul:new(get_opts(consul_server, true))
+
+    ::RETRY::
+    local watch_result, watch_err = client:get(consul_server.consul_watch_catalog_url)
+    local watch_error_info = (watch_err ~= nil and watch_err)
+            or ((watch_result ~= nil and watch_result.status ~= 200)
+            and watch_result.status)
```

Comment:
```suggestion
    local watch_error_info = (watch_err ~= nil and watch_err)
                             or ((watch_result ~= nil and watch_result.status ~= 200)
                             and watch_result.status)
```
Plz follow the [CODE_STYLE](https://github.com/apache/apisix/blob/master/CODE_STYLE.md#maximum-length-per-line)

Reply:
done